caikit/runtime/train.py

```diff
@@ -65,7 +66,8 @@ def error(self, message):
         raise ArgumentParserError(f"{self.prog}: error: {message}")
 
 
-def write_termination_log(text: str, log_file="/dev/termination-log"):
+def write_termination_log(text: str):
+    log_file = os.environ.get("TERMINATION_LOG_FILE", "/dev/termination-log")
```

Comment:
I probably missed this in a previous PR, but I'd like to make the termination log stuff configurable (and optional). This should be managed via a command line arg like the rest of the configuration. I'd also support something more clever that allows the CLI args to be set via env var as well, but I'd like to avoid `os.environ` as much as possible unless it's systematic.

Reply:
Got you. I did it this was since this was also how we are doing it for fine tuning, so I was trying to keep the configs consistent. Open to changing it to be an argument like the other parameters.
If it can't write to the termination log, it just catches the exception and moves on. That's kind of how I made it "optional" but can make it more explicitly optional if that makes sense. Or at least turned on by default and you can disable it if you like.

Reply:
You can do the env + cli thing as follows:

```
parser.add_argument("-f", "--foo", default=os.getenv("FOO", "default-foo"), ...)
```